plugin/trino-prometheus/src/test/java/io/trino/plugin/prometheus/TestPrometheusIntegrationMetrics.java

```diff
@@ -100,7 +101,7 @@ public void testGetColumnMetadata()
         // directly.
     }
 
-    @Test(expectedExceptions = TrinoException.class, dependsOnMethods = "testRetrieveUpValue")
+    @Test(expectedExceptions = TrinoException.class)
```

Comment:
as a followup we should remove `expectedExceptions` use and replace it with `assertThatThrownBy`